samples/manager/manager.js

```diff
@@ -449,7 +449,14 @@ const listDevices = async (registryId, projectId, cloudRegion) => {
       registryId
     );
 
-    const [response] = await iotClient.listDevices({parent: parentName});
+    // See full list of device fields: https://cloud.google.com/iot/docs/reference/cloudiot/rest/v1/projects.locations.registries.devices
+    // Warning! Use snake_case field names.
+    const fieldMask = { paths: [ 'id', 'name', 'num_id', 'credentials', 'last_heartbeat_time', 'last_event_time', 'last_state_time', 'last_config_ack_time', 'last_config_send_time', 'blocked', 'last_error_time', 'last_error_status', 'config', 'state', 'log_level', 'metadata', 'gateway_config' ] };
```

Comment:
This is definitely a place where I would break this line up, for example:

```
const fieldMask = {
    paths: [
        'id',
        'name',
        ...
    ]
};
```

You can probably use `prettier` to automatically format it for you.